### DISCUSSION TITLE
Issue : Exit value 0 and results file not found

### DISCUSSION THREAD

**Discussion by justinkirion** [reach-lab/MixWildGUI#167] [Help Wanted]
Hello there, 

I come up with a new problem to solve. With one of the models I want to compute, when I launch MixWild Stage 1, the algorithm iterations don't start. I end up with an Exit Value of 0, and two error messages appear : the first one just contains "Failed", and the second one says : "[file path]\[data].stage1.out (The specified file can not be found)". Do you know in which cases this type of error could happen ?
I join a screenshot to illustrate my point. 

Thanks for your previous answers and thanks in advance for this one !
Justin
![Github3](https://user-images.githubusercontent.com/64084594/176141312-03503bb7-1a58-46fd-8908-93aeb3a2a4a9.PNG)

**Comment by rexli999** [comment]
Hi Justin, in the emails we exchanged before, you were able to solve this issue by using the latest version of MixWILD. I just commented it here for others' reference.
